lib/export.js

```diff
@@ -847,6 +831,17 @@ class FHIRExporter {
       return;
     }
 
+    // Insert mapping rules into the FHIR mapping element
+    ss.mapping = ss.mapping || [];
+    const isValue = sourceValue.effectiveIdentifier == def.value.identifier;
```

Comment:
Value can be undefined.  Add something like this `def.value !== undefined &&`

Reply:
Good catch. Updating now.